src/views/cards/cards.jsx

```diff
@@ -52,7 +52,7 @@ var Cards = injectIntl(React.createClass({
                                     <FormattedMessage id='cards.viewCard' />
                                     {(
                                         formattedLinks['cards.starterLink'] === englishLinks['cards.starterLink'] &&
-                                        locale !== 'en'
+                                        locale.indexOf('en') === -1
```

Comment:
👍 on generalizing. Just fyi, I switched to using `this.props.intl.locale` to avoid having to clean it like you suggest in `locale.split('_')[0] !== 'en'`, since we already to that in `render`. It doesn't appear that the value in `this.props.intl.locale` could ever be something that includes `en` otherwise. I guess switching does prevent against adding future locales that would have that though